File: java_backup/my java/bishwarup_!/armstrong.java
class armstrong
{
public static void main(int n)
{
int n1,q,r,s=0;
n1=n;
while(n>999)
{
q=n/10;
r=n%10;
s=s+(r*r*r);
n=q;
}
if(n1==s)
System.out.print(n1+" is armstrong");
else
{
System.out.print(n1+"not armstrong");
}
}
}

File: 1st_and_2nd_Sem_c_backup/BIN/C Prog/practice c/armstrong.c
#include<stdio.h>
int main()
{
	int n,temp,num=0;
	scanf("%d",&n);
	temp=n;
	while(temp!=0)
	{
		num=num+pow(temp%10,3);		
		temp=temp/10;
	}
	if(num==n)
		printf("armstrong");
	else
		printf("not armstrong");
		return 0;		
}
